services/checks/checks.go

```diff
@@ -70,6 +71,11 @@ const (
 	maxSupportedVersion = 1
 )
 
+var (
+	pmmAgentVersionTwoDotSix   *version.Parsed
+	pmmAgentVersionTwoDotSeven *version.Parsed
+)
```

Comment:
Yes, it's ugly, but the other option was `pmmAgentVersion_2_6_7`

Reply:
I'm pretty sure there are more options :)

Reply:
My imagination has run out for today 😔. May be tomorrow I will find better names.